### DISCUSSION TITLE
Format Query Button is grayed out

### DISCUSSION THREAD

**Discussion by ShaynaFishman** [DaxStudio/DaxStudio#784] [Q&A]
I can't figure out how to make the Format Query button work. 
I understand the formatter button uses daxformatter.com to format queries and therefore you need active internet connection to use it. My computer has an active internet connection. Is there some place I have to specifically connect DAXStudio to the internet?

If not, what else might be causing the button to be grayed out?

**Comment by dgosbell** [comment]
Assuming that you have a document open, the only thing that can cause the format query button to be disabled is if you have switched on either **Block All Internet Access** or **Block External Services** in the privacy options

![image](https://user-images.githubusercontent.com/8241830/147802940-8c0b29e4-def7-4b11-9e0a-76efab32819c.png)
